client/service/src/client/call_executor.rs

```diff
@@ -125,10 +125,12 @@ where
 		strategy: ExecutionStrategy,
 		extensions: Option<Extensions>,
 	) -> sp_blockchain::Result<Vec<u8>> {
-		let mut changes = OverlayedChanges::default();
+		let mut changes = 
 		if self.client_config.offchain_indexing_api {
```

Comment:
```suggestion
		let mut changes = if self.client_config.offchain_indexing_api {
```
That's a pretty confusing line break :)